### DISCUSSION TITLE
Merged to master : Custom domain no longer works

### DISCUSSION THREAD

**Discussion by RossHolland-Melt** [vercel/community#367] [Help] [answered]
I merged my code to the master branch and deployed : My custom domain registers as being fine but it gives ma connection issue which is not isolated to my devices.  The vercel provided project domain still works fine  : https://project-handmade.vercel.app/

Custom domain : abenziafrica.uk and www.abenziafrica.uk

**Comment by samcx** [comment, marked as answer]
Apologies for the late response! It looks like you were able to resolve the issue. Let us know if you have any further questions or concerns.
